File: csharp/VS2008/netframework/Modules/10.API/25.Encrypted Files/PasswordDialog.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

namespace EncryptedFiles
{
    /// <summary>
    /// Summary description for PasswordDialog.
    /// </summary>
    public partial class PasswordDialog: System.Windows.Forms.Form
    {

        public PasswordDialog()
        {
            //
            // Required for Windows Form Designer support
            //
            InitializeComponent();

            //
            // TODO: Add any constructor code after InitializeComponent call
            //
        }

        public string Password
        {
            get
            {
                return PasswordEdit.Text;
            }
        }
    }
}

File: vb/VS2005/netframework/Modules/10.API/25.Encrypted Files/PasswordDialog.vb
Imports System.Collections
Imports System.ComponentModel

Namespace EncryptedFiles
	''' <summary>
	''' Summary description for PasswordDialog.
	''' </summary>
	Partial Public Class PasswordDialog
		Inherits System.Windows.Forms.Form

		Public Sub New()
			'
			' Required for Windows Form Designer support
			'
			InitializeComponent()

			'
			' TODO: Add any constructor code after InitializeComponent call
			'
		End Sub

		Public ReadOnly Property Password() As String
			Get
				Return PasswordEdit.Text
			End Get
		End Property
	End Class
End Namespace
